pipelines/kukkuu-api-dev.yml

```diff
@@ -41,15 +37,13 @@ resources:
       name: kukkuu/kukkuu-pipelines
 
 extends:
-  # Filename in Azure DevOps Repository (note possible -ui or -api)
-  # Django example: azure-pipelines-PROJECTNAME-api-release.yml
-  # Drupal example: azure-pipelines-drupal-release.yml
-  template: azure-pipelines-kukkuu-api-test.yml@kukkuu-pipelines
+  # Filename in Azure DevOps Repository
+  template: components/kukkuu-api/pipelines/kukkuu-api-dev.yml@kukkuu-pipelines
   parameters:
     # Application build arguments and config map values as key value pairs.
-    # Does not contain all buildArguments, the rest located in kukkuu-pipelines/kukkuu-variables-development.yml and kukkuu-variables-common.yml
+    # Does not contain all buildArguments or configMap values, the rest located in kukkuu-pipelines
     # The values here will override the values defined in the kukkuu-pipelines repository
     # buildArgs:
     configMap: # pod environment variables
       TOKEN_AUTH_BROWSER_TEST_ENABLED: 1
-      TOKEN_AUTH_BROWSER_TEST_JWT_ISSUER: https://kukkuu-ui.test.hel.ninja,https://kukkuu-admin.test.hel.ninja
+      TOKEN_AUTH_BROWSER_TEST_JWT_ISSUER: https://kukkuu-ui.dev.hel.ninja,https://kukkuu-admin.dev.hel.ninja
```

Comment:
Browser testing uses a shared secret with JWT, so we can get rid of any real authorization service. So, the public UI and the admin UI actually generates a JWT by themselves. The JWT is attached to authorization header of HTTP request done to API. The API then needs to validate the JWT issuer. `TOKEN_AUTH_BROWSER_TEST_JWT_ISSUER` is the environment variable that defines the issuer, so it needs to match to the one that is set in client.

I think that we have been running the browser tests in dev (a PR environment) and test environments. An UI dev server has been connected to a test API, so a test-domain has been a good issuer name for both.

Reply:
This will change, UI dev servers will be connected to a dev API instead of test API